projects/rero/ng-core/src/lib/record/editor/widgets/label/label.component.html

```diff
@@ -0,0 +1,80 @@
+<!--
+ RERO angular core
+ Copyright (C) 2020 RERO
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, version 3 of the License.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="field">
+  <!-- header and menu -->
+  <ng-container *ngIf="!isRoot()">
+    <ng-container *ngTemplateOutlet="hasMenu(field) ? menu : title; context: { f: field }">
+    </ng-container>
+  </ng-container>
+
+  <!-- TEMPLATES -->
+  <!-- dropdown menu -->
+  <ng-template #menu let-f="f">
+    <div class="header">
+      <ng-core-editor-dropdown-label-editor [field]="f" [canAdd]="false">
+        <ng-container dropdown-list>
+          <ng-container *ngFor="
+          let fChildren of hiddenFieldGroup(getFieldGroup(f));
+          let first = first;
+          let last = last
+        ">
+            <!-- header -->
+            <h6 *ngIf="first" class="dropdown-header" translate>Add fields</h6>
+            <!-- dropdown items -->
+            <button class="dropdown-item" (click)="show(fChildren)" type="button">
+              {{ translateLabel(fChildren) | async }}
+            </button>
+            <div *ngIf="last && f.templateOptions.helpURL" class="dropdown-divider"></div>
+          </ng-container>
+          <!-- help link -->
+          <a *ngIf="f.templateOptions.helpURL" class="dropdown-item" [href]="f.templateOptions.helpURL" translate>
+            Help
+          </a>
+        </ng-container>
+        <ng-container buttons>
+          <button type="button" *ngIf="canAdd()" (click)="add()" class="btn btn-link text-secondary btn-sm">
+            <i class="fa fa-clone"></i>
+          </button>
+          <!-- hide button -->
+          <button type="button" (click)="remove()" *ngIf="canRemove()"
+            class="btn btn-link text-secondary btn-sm align-baseline">
+            <i class="fa fa-trash"></i>
+          </button>
+        </ng-container>
+      </ng-core-editor-dropdown-label-editor>
+
+    </div>
+  </ng-template>
+
+  <!-- section title -->
+  <ng-template #title let-f="f">
+    <div class="header">
+      <label class="mb-0" *ngIf="f.templateOptions.label" [attr.for]="f.id" [tooltip]="f.templateOptions.description">
```

Comment:
maybe introduce the possibility to add classes on label ? 
See https://github.com/rero/ng-core/pull/339/files#diff-869d70bf847d429975de17d5bba4a3b9d16c3ce727a971e147a93deaaa8d0fcdR25

Reply:
I prefer the more generic approach: https://github.com/rero/ng-core/pull/338/files#diff-869d70bf847d429975de17d5bba4a3b9d16c3ce727a971e147a93deaaa8d0fcdR24